packages/taro-runtime/src/dom/element.ts

```diff
@@ -105,13 +105,19 @@ export class TaroElement extends TaroNode {
   }
 
   public removeAttribute (qualifiedName: string) {
+    const isVue = process.env.FRAMEWORK === 'vue'
+    const internalComponent = internalComponents[toCamelCase(capitalize(this.tagName.toLowerCase()))]
```

Comment:
这里应该移动到 `// avoid attribute being removed because set false value in vue` 的分支里去，我们不希望在非 vue 的情况也执行这个语句和其中包含的函数。
另外先 toCamelCase 再 capitalize 比较稳健，因为 capitalize 已经没有优化的空间了，toCamelCase 以后可能还会改。